scripts/update_dependencies.sh

```diff
@@ -0,0 +1,34 @@
+set -e
+
+if [ "$1" ]
+  then
+    REQ_BRANCH="$1"
+else
+    REQ_BRANCH="develop"
+fi
```

Comment:
I see your point, and I sort of agree. The reason I had it like this was so that everyone is then always on the most recent dependencies irrelevant of branch, essentially how it will be when we have docker images. 

I was thinking for versioned releases I would modify this to the version tag.

I do agree its a bit counter intuitive but I dont want people o be on different dependency versions...

Reply:
I guess it depends on what the purpose of the script is. If it's just so that we don't need to remember the "update" commands, I would say we should use the current branch by default. If the purpose is to get the most up-to-date dependencies from the repo, then we can stick with this.

I prefer the former, but am happy to be overruled.

Reply:
I dont really mind @CoronelBuendia which would you prefer :slightly_smiling_face:?

Reply:
Mmm.. to be discussed in our meeting later perhaps. Not sure either. Really depends on how often we're changing our dependencies. I'm on the fence at the moment